Remotion/ObjectBinding/Web.Development.WebTesting.TestSite/App.config

```diff
@@ -16,7 +16,11 @@
 
   <system.web>
     <globalization culture="de-AT" uiCulture="en-US"/>
-    <compilation debug="true" />
+    <compilation debug="true">
+      <assemblies>
+        <add assembly="System.Drawing.Primitives"/>
```

Comment:
Is this really needed or an artifact from another branch?

Reply:
Seems like the date picker popup is broken in .NET because of this. I am going to create a bug ticket for it.